app/models/org.rb

```diff
@@ -60,6 +60,10 @@ class Org < ActiveRecord::Base
 
   # Predefined queries for retrieving the managain organisation and funders
   scope :managing_orgs, -> { where(abbreviation: Rails.configuration.branding[:organisation][:abbreviation]) }
+  scope :funders, -> { where(org_type: 2) }
```

Comment:
Please remove these 2 scopes. They are provided already by the FlagShiTzu gem. I had removed them in an earlier PR.